apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -451,6 +458,21 @@ class ManageStudentsTable extends Component {
         selectedColumn,
       }),
     });
+    if (selectedColumn === COLUMNS.FAMILY_NAME) {
+      this.props.setSortByFamilyName(
+        'true',
```

Comment:
oh, or were you intentionally using strings? is that a pattern we use?

Reply:
I was noticing that somewhere during the dispatch, it was getting converted anyway so was just trying to match that convention. Not sure why it was happening, and can try to get them converted back!

Reply:
Modified!

Reply:
Thank you! 🙏